app/views/internal/users/show.html.erb

```diff
@@ -101,14 +101,17 @@
 </div>
 
 <% if @user.seeking_mentorship %>
+  <h2> Mentee Description </h2>
+        <p style="position:sticky;top:50px;background:white;z-index:1000;padding:10px"><%= @user.mentee_description %></p>
+
   <h2> Possible Mentors </h2>
   <% count = 0 %>
-  <% User.where(offering_mentorship: true).order("mentor_form_updated_at DESC").each do |possible_mentor| %>
-    <% break if count > 18 %>
+  <% User.where(offering_mentorship: true).where.not(mentee_form_updated_at: nil).order("mentor_form_updated_at DESC").each do |possible_mentor| %>
```

Comment:
This should be `menTOR_form_updated_at` right?

Reply:
Yeah good call